02-peer-protocol.md

```diff
@@ -420,16 +412,91 @@ it is possible for the fee for the exchanged witness data to be underpaid.
 It is the responsibility of the sending peer to correctly account for the
 required fee, e.g. a multisig witness stack whose weight exceeds 110.
 
+### The `tx_init_rbf` Message
+
+This message initiates a replacement of the transaction after it's been
+completed.
+
+1. type: 72 (`tx_init_rbf`)
+2. data:
+   * [`channel_id`:`channel_id`]
+   * [`u32`:`locktime`]
+   * [`u32`:`feerate`]
+
+1. `tlv_stream`: `tx_init_rbf_tlvs`
+2. types:
+    1. type: 0 (`shared_output_contribution`)
```

Comment:
```suggestion
    1. type: 0 (`funding_output_contribution`)
```

Reply:
Updated in https://github.com/niftynei/lightning-rfc/pull/3/commits/37491624085d967f9a36372d9112d1d3589607e5